tests/unittest_brain_numpy_ma.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021 hippo91 <guillaume.peillex@gmail.com>
+
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+import pytest
+
+try:
+    import numpy  # pylint: disable=unused-import
+
+    HAS_NUMPY = True
+except ImportError:
+    HAS_NUMPY = False
+
+from astroid import builder
+
+
+@pytest.mark.skipif(not HAS_NUMPY, reason="This test requires the numpy library.")
```

Comment:
```suggestion
@pytest.mark.skipif(HAS_NUMPY is False, reason="This test requires the numpy library.")
```
Technically the same. Might be just personal preference as this is how I think of it.